### ISSUE TITLE
Example for Stepper

### ISSUE THREAD

**Issue by Jul3k** [mariobuikhuizen/ipyvuetify#48]
I am trying to get a bit more complex stepper component working with help of a template by the following code:
```
import ipyvuetify as v
import ipywidgets as widgets
from traitlets import (Unicode, List, Int, Bool, Any, Dict)

test_component = v.Select(items=['identity', 'log', 'log10', 'log1p', 'log1p'])

class Stepper(v.VuetifyTemplate):
    page = Any(1).tag(sync=True)
    items = List([{'title':"Title 1", 'content':test_component},
                  {'title':"Title 2", 'content':'test content'},
                  ]).tag(sync=True, **widgets.widget_serialization)
    

    template=Unicode("""
    <v-stepper v-model="page" vertical>
    <template v-for="(item, index) in items">
      <v-stepper-step :complete="page > index+1" :step="index+1">
        {{item.title}}
      </v-stepper-step>
      <v-stepper-content :step="index+1">
        <v-card color="grey lighten-5" class="mb-12" height="200px">
        <{{item.content}}/>
        </v-card>
        <v-btn v-if="index<items.length-1" color="primary" @click="page = index+2">Continue</v-btn>
        <v-btn v-if=index text @click="page = index">Back</v-btn>
      </v-stepper-content>
      </template>
    </v-stepper>
    """).tag(sync=True)
    
Stepper()
```

I could not figure out how to get a already existing component inserted into the template. Is this the correct way or am I completely off track. The stepper only displays the component as text:
<IPY_MODEL_f450711f42734468a93c3d6392e33cb2/>
![image](https://user-images.githubusercontent.com/1540387/72922178-e19a0e00-3d4c-11ea-80c1-f78a18ff90d5.png)

I would be very graceful if somebody could point me in the right direction or tell me if this is not possible in the way I imagine it. If there is a better way to do this please let me know and an example would be very nice.

**Comment by Jul3k** [comment]
I finally got it working with a mix of python and vuejs templates. Not sure if this is the most elegant solution to the task. I am defenitly not a javascript expert :joy: 
```
pages = [{'title':"Title 1", 'content':test_component1},
        {'title':"Title 2", 'content':test_component2}]

def MyStepper(pages):
    ftemplate = '<template><v-stepper v-model="page" vertical>'
    for index,p in enumerate(pages, start=1):
        ftemplate += f"""
          <v-stepper-step :complete="page > {index}" :step="{index}">
            {p['title']}
          </v-stepper-step>
          <v-stepper-content :step="{index}">
            <v-card color="grey lighten-5" class="mb-12" height="200px">
             <cc{index}/>
            </v-card>
            <v-btn v-if="{index}<{len(pages)}" color="primary" @click="page = {index+1}">Continue</v-btn>
            <v-btn v-if={index-1} text @click="page = {index-1}">Back</v-btn>
          </v-stepper-content>
        """
    ftemplate += '</v-stepper>'
    ftemplate += '</template>'
    
    class Stepper(v.VuetifyTemplate):
        page = Any(1).tag(sync=True)
        template=Unicode(ftemplate).tag(sync=True)
        
    return Stepper(components = {f'cc{index}':p['content'] for index,p in enumerate(pages, start=1)})

MyStepper(pages)
```

**Comment by mariobuikhuizen** [comment]
The current solution for embedding other widgets in a template is very limited and different from vue.

Your solution is very clever! But ideally it should not be necessary to template the template :wink:.

This issue gave me an idea on how to solve this in the ipyvue library. Not sure if it's possible, I will get back on that.

**Comment by Jul3k** [comment]
Yes, the template in a template idea was born in deep desperation. I hope you find a more elegant way. Really like this project so far. Good work :)

**Comment by MichaMucha** [comment]
Hey @Jul3k looks like @mariobuikhuizen built it! :)
https://github.com/mariobuikhuizen/ipyvue/pull/4

Thank you!

**Comment by mariobuikhuizen** [comment]
I had lost sight of this issue, thanks @MichaMucha for picking this up!

mariobuikhuizen/ipyvue#4 has also been released as ipyvue 1.1.0 on Pypi and conda.

**Comment by Jul3k** [comment]
Great work. Thank you :)
